content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -70,12 +70,7 @@ The HorizontalPodAutoscaler normally fetches metrics from a series of aggregated
 `custom.metrics.k8s.io`, and `external.metrics.k8s.io`).  The `metrics.k8s.io` API is usually provided by
 metrics-server, which needs to be launched separately. See
 [metrics-server](/docs/tasks/debug-application-cluster/resource-metrics-pipeline/#metrics-server)
-for instructions. The HorizontalPodAutoscaler can also fetch metrics directly from Heapster.
-
-{{< note >}}
-{{< feature-state state="deprecated" for_k8s_version="v1.11" >}}
-Fetching metrics from Heapster is deprecated as of Kubernetes 1.11.
-{{< /note >}}
+for instructions.
```

Comment:
Hi @Shubham82 . Thanks for this PR.
The preview looks good, though, what do you think about adding more descriptive link?

"See [metrics-server](...) for instructions."

For more information about resource metrics, see [Metrics Server](...).

Reply:
Thanks, @kbhawkey, for looking into this.

> For more information about resource metrics, see Metrics Server.

this seems good to me.  Adding this instead of  `See metrics-server for instructions.`  is looking better.

I will update this soon.

Reply:
Hi @kbhawkey, 
Addressed the comment Please take a look.

Reply:
Hi @kbhawkey  @tengqm  
can you please take a look, and if it seems good can we merge this pr?